docs/how-to/deploy-ck8s-aws.rst

```diff
@@ -0,0 +1,68 @@
+Deploy Charmed MLflow to Charmed Kubernetes on AWS
+========================================================
+
++------------+---------+
+| Component  | Version |
++============+=========+
+| MLflow     | 2       |
++------------+---------+
```

Comment:
This is not rendered nicely, I think these should be centred. Would something like this do?
```suggestion
+-----------+---------+
| Component | Version |
+===========+=========+
|   MLflow  |    2    |
+-----------+---------+
```

This applies to the rest of the files as well.

Reply:
Yeah I played around with this before trying to get it centred but couldn't. Not a big deal. We may end up going with a different versioning system so don't wanna spend too much time on it.

Reply:
@ColmBhandal, the previous suggestion won't work indeed, but replacing `=` with `-` actually helps a lot with the indentation.
```suggestion
+-----------+---------+
| Component | Version |
+-----------+---------+
|   MLflow  |    2    |
+-----------+---------+
```
![image](https://github.com/canonical/mlflow-operator/assets/57714042/13e29808-b098-4cef-867f-602b11e31fa2)

Reply:
Good point. I updated all tables following your suggestion.